### PR TITLE
Generate notifications for bidding actions

### PR DESCRIPTION
**Pull request by mjoyce91**
When a bureau extends or revokes:
- generate notification for bidder

When a CDO accepts or declines:
- generate notification for bidder and bureau

When a bidder accepts or declines:
- generate notification for bureau

### CODE REVIEW

File: talentmap_api/bidding/views/bidhandshake.py [hunk 1]
```diff
@@ -78,6 +80,7 @@ def delete(self, request, pk, cp_id, format=None):
         else:
             user = UserProfile.objects.get(user=self.request.user)
```
Comment (elizabeth-jimenez) [inline talentmap_api/bidding/views/bidhandshake.py:81]: I know this isn't from this PR, but just realized that line 81 === line 75 🧼

File: talentmap_api/common/common_helpers.py [hunk 2]
```diff
@@ -488,4 +488,43 @@ def send_email(subject = '', body = '', recipients = []):
...
+
+
+def cdoHandshakeNotification(perdet, cp_id, is_accept=True):
+    from talentmap_api.user_profile.models import UserProfile
+    from talentmap_api.bidding.models import BidHandshake
+    action = "accepted" if is_accept else "declined"
+    user = UserProfile.objects.get(emp_id=perdet)
+    bureau = BidHandshake.objects.get(cp_id=cp_id).owner
+    if user:
+        message = f"CDO has {action} handshake on behalf of you for position with ID {cp_id}"
```
Comment (elizabeth-jimenez) [inline talentmap_api/common/common_helpers.py:508]: wondering if "on your behalf for position" may sound better?

**Comment by elizabeth-jimenez** [comment]
When a bureau extends or revokes:
- [x] extends: generate notification for bidder
- [x] Revokes: generate notification for bidder
When a CDO accepts or declines:
- [x] acepts: generate notification for bidder 
- [x] declines: generate notification for bidder 
- [x] accepts: and bureau
- [x] declines: and bureau
When a bidder accepts or declines:
- [x] Accept: generate notification for bureau
- [x] Decline: generate notification for bureau

**Review by elizabeth-jimenez — APPROVED** [review]
🧁 🍰 🍬 👍

**Comment by elizabeth-jimenez** [comment]
Looks good, just left a few comments, but not stoppers

File: talentmap_api/common/common_helpers.py (hunk 2 above)
Comment (mjoyce91) [reply talentmap_api/common/common_helpers.py]: it does

**Pull request merged**